platform/lib/pipeline/componentReferenceImporter.js

```diff
@@ -15,289 +15,220 @@
  */
 require('module-alias/register');
 
-// By default, reference docs are imported from the latest release tag.
-// If a doc is broken in a release, add it to this this list to fetch from master instead.
-//
-// DON'T FORGET TO REMOVE ONCE IT'S FIXED
-const DOCS_TO_FETCH_FROM_MASTER = ['amp-script', 'amp-carousel'];
```

Comment:
Where did this go?

Reply:
Moved to a configuration file to be aligned with the behaviour of SpecImporter: platform/config/imports/componentReference.json